src/globus_sdk/experimental/auth_requirements_error/utils.py

```diff
@@ -56,18 +57,18 @@ def to_auth_requirements_error(
     # Prefer a proper auth requirements error, if possible
     try:
         return GlobusAuthRequirementsError.from_dict(error_dict)
-    except ValueError as err:
+    except _validators.ValidationError as err:
```

Comment:
`ValidationError` inherits from `ValueError`. Can the base `ValueError` be raised by `.from_dict()`?

Reply:
My take is that I'd like us to have a specific error class which means "validation failed".
Any case of a base `ValueError` _should_, in that line of thinking, mean that something which we didn't control failed.

Since the `ValidationError` could be raised into a caller-visible context, I also think I need to add it to the exported names in `__init__.py` and move it out of a purely-internal module (optional, but IMO cleaner).

Reply:
Scratch that. I'm having trouble naming the module. I'm just going to expose it in `__init__` from the current location.